src/status.ts

```diff
@@ -0,0 +1,8 @@
+function onChange(tasks: any, position: string): number[] {
+  console.log(tasks)
+  console.log(tasks[parseInt(position)])
+  tasks[parseInt(position)].completed = !tasks[parseInt(position)].completed
+  return tasks
+}
+
+export { onChange }
```

Comment:
- [ ] [OPTIONAL]You need to use plain JS, TypeScript is a no-no in this part of the program, also, don't commit any console.log